custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
-                            "x-ms-summary": "message"
-                          },
-                          "initiatorCommunicationIdentifier": {
-                            "$ref": "#/definitions/CommunicationIdentifier"
-                          },
-                          "topic": {
-                            "type": "string",
-                            "description": "Chat message content for messages of type topicUpdated.",
-                            "x-ms-summary": "Topic"
-                          },
-                          "participants": {
-                            "type": "array",
-                            "items": {
-                              "type": "object",
-                              "properties": {
-                                "communicationIdentifier": {
-                                  "$ref": "#/definitions/CommunicationIdentifier"
-                                }
-                              }
-                            },
-                            "description": "Chat message content for messages of types participantAdded or participantRemoved."
-                          }
-                        },
-                        "description": "Content of a chat message."
-                      },
-                      "senderDisplayName": {
-                        "type": "string",
-                        "description": "Display name of the chat message sender. This property is used to populate sender name for push notifications.",
-                        "x-ms-summary": "Sender Display Name"
-                      },
-                      "createdOn": {
-                        "type": "string",
-                        "description": "Timestamp in RFC3339 format (yyyy-MM-ddTHH:mm:ssZ) when the chat message arrived at the server.",
-                        "x-ms-summary": "Created On"
-                      },
-                      "senderCommunicationIdentifier": {
-                        "$ref": "#/definitions/CommunicationIdentifier"
-                      }
-                    }
+                    "$ref": "#/definitions/Contact"
                   }
-                },
-                "nextLink": {
-                  "type": "string",
-                  "description": "If there are more chat messages that can be retrieved, the next link will be populated.",
-                  "x-ms-summary": "Next Link"
                 }
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "summary": "List chat thread messages",
-        "description": "List messages From chat thread.",
-        "operationId": "ListMessages",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/ListMessages